review_practice/react-app/src/containers/SearchBar/ModalBody/PriceFilter.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useContext } from 'react';
+
+import { SearchRoomContext } from 'pages/SearchRoom';
+
+import PriceRangeSlider from 'components/PriceRangeSlider';
+
+const MIN_PRICE = 0;
+const MAX_PRICE = 100000;
+
+export default ({ buttonState, setButtonState }) => {
+    // Context
+    const {
+        searchOption: {
+            searchOptions: { price },
+        },
+        dispatchSearchOption,
+    } = useContext(SearchRoomContext);
+
+    // State
+    const priceState = useState([
+        price && price.startPrice ? price.startPrice : MIN_PRICE,
+        price && price.endPrice ? price.endPrice : MAX_PRICE,
```

Comment:
optional chaining 을 활용하면 아래와 같이 표현가능합니다.
```suggestion
        price?.endPrice ? price.endPrice : MAX_PRICE,
```